### ISSUE TITLE
train on custom dataset

### ISSUE THREAD

**Issue by SantiDiazC** [stereolabs/zed-pytorch#2]
Hi I'm working on a deep reinforcement project using mask Rcnn algorithm. Previously I used the tensorflow object detection api example provided in the documentation to train on my custom dataset. However, I decided to move to pytorch so I wondered if there is a tutorial or documentation on how to set the mask Rcnn algorithm provided here on your custom dataset. thanks in advance.
